doc: add documentation for uninstallation using GitOps tools (#902)
longhorn/website: content/docs/1.6.0/deploy/uninstall/_index.md

helm uninstall longhorn -n longhorn-system
```

> ### Uninstalling Longhorn using Fleet
> 
> Run this command:

Reviewer: ```suggestion
### Uninstalling Longhorn Using Fleet

Run the following command:
```

Author: Refined. Thank you!

```
kubectl delete GitRepo longhorn -n fleet-local
```

> After the `longhorn-uninstall` job is done, run this command:

Reviewer: ```suggestion
After the `longhorn-uninstall` job is completed, run the following command:
```

Author: Refined. Thank you!

```
kubectl delete -f https://raw.githubusercontent.com/longhorn/longhorn/v{{< current-version >}}/deploy/longhorn.yaml
```

> ### Uninstalling Longhorn using Flux
> 
> Run this command:

Reviewer: ```suggestion
### Uninstalling Longhorn Using Flux

Run the following command:
```

Author: Refined. Thank you!

```
flux delete helmrelease longhorn-release -n longhorn-system
```

> ### Uninstalling Longhorn using Argo CD
> 
> Currently, the PreDelete resource hook is not supported in Argo CD. So, instead of running `argocd app delete longhorn` directly, manually [uninstall Longhorn using kubectl](#uninstalling-longhorn-using-kubectl) to prevent dangling resources left over in longhorn-system namespace.

Reviewer: ```suggestion
### Uninstalling Longhorn Using Argo CD

Argo CD currently does not support the PreDelete resource hook. Instead of running `argocd app delete longhorn` directly, you must [uninstall Longhorn using kubectl](#uninstalling-longhorn-using-kubectl) to prevent dangling resources from remaining in the `longhorn-system` namespace.
```

Author: Refined. Thank you!
